.gitlab-ci.yml

```
@@ -46,16 +46,19 @@ deploy_to_sonatype:
   tags:
     - "runner:docker"
 
-  image: eclipse-temurin:8u442-b06-jdk
+  image: *jdk-image
 
   script:
     # Ensure we don't print commands being run to the logs during credential
     # operations
     - set +x
 
-    - echo "Installing AWSCLI..."
+    - echo "Setting up Python virtual environment..."
     - apt update
-    - apt install -y python3 python3-pip
+    - apt install -y python3 python3-pip python3-venv
+    - python3 -m venv venv
+    - source venv/bin/activate
+    - python3 -m pip install --upgrade pip
     - python3 -m pip install awscli
```

We may still want this bit...

```suggestion
    - echo "Installing AWSCLI..."
    - python3 -m pip install awscli
```